tests/test_scripts.py

```diff
@@ -94,21 +98,36 @@ def test_train_adversarial(tmpdir):
         "log_root": tmpdir,
         "rollout_path": "tests/data/expert_models/cartpole_0/rollouts/final.pkl",
         "init_tensorboard": True,
-        "plot_interval": 1,
-        "extra_episode_data_interval": 1,
     }
-    run = train_ex.run(named_configs=named_configs, config_updates=config_updates,)
+    run = train_adversarial.train_ex.run(
+        named_configs=named_configs, config_updates=config_updates,
+    )
     assert run.status == "COMPLETED"
     _check_train_ex_result(run.result)
 
 
+def test_train_adversarial_algorithm_config_error(tmpdir):
+    """Error on bad algorithm arguments."""
+    named_configs = ["cartpole", "fast"]
+    config_updates = {
+        "log_root": tmpdir,
```

Comment:
Why are these additional config updates necessary (besides the `"BAD_VALUE"`)?

Reply:
"init_tensorboard" is not necessary (removing this), but "rollout_path" (expert_models not loaded on test server IIRC) and "log_root" (don't clog my logs) are.